docs/dasharo-tools-suite/overview.md

```diff
@@ -24,7 +24,35 @@ Thank you for using Dasharo Tools Suite. If you have encountered any problems
 with this system or would like to provide feedback for us - please open an issue
 on [Dasharo
 issues](https://github.com/Dasharo/dasharo-issues/issues?q=is%3Aopen+is%3Aissue+label%3ADasharoToolsSuite).
+Dasharo Tools Suite has its own label: `DasharoToolsSuite` and issue template
+for bugs: `DTS bug report`. Other types of issues in the repository are generic
+and can be used with the `DasharoToolsSuite` label.
 
 And if you have already used this system and would be interested in supporting
 the project, please check the
 [How to support us?](../osf-trivia-list/dts.md#how-to-support-us) section.
+
+## Gathering additional information and providing logs
+
+It is not always possible to copy/paste logs from DTS. In such cases, photos or
+screenshots are also acceptable. Just imagine you are a professional
+photographer trying to capture another natural miracle in the best quality
+possible.
+
+There is a list of common tools which can be used to get more information about
+your case:
+
+* `dmidecode`: will provide detailed information about your system;
```

Comment:
I am not sure in which cases these commands would be useful when reporting bugs agains DTS. Have we looked at some past bugs to see which information was missing? If that would be required, it should go into template, but I am not so convinced.

Reply:
It is useful if some extra info is needed while reporting an issue, but I guess it is issue specific and should not be added to this documentation. Amended here b07c88d5d094dcab8b4de9bbec664d15942e09fc.